cmd/stayrtr/stayrtr.go

```diff
@@ -255,6 +255,11 @@ func (e IdenticalFile) Error() string {
 func (s *state) updateFromNewState() error {
 	sessid := s.server.GetSessionId()
 
+	vrpsjson := s.lastdata.Data
+	if (vrpsjson == nil) {
+		return nil
```

Comment:
I feel that we should probably return an error here. I need to look at the complete structure of the object we have here though.